thirdeye/thirdeye-pinot/src/main/java/org/apache/pinot/thirdeye/anomaly/monitor/MonitorTaskRunner.java

```diff
@@ -64,9 +64,19 @@
 
   private void executeMonitorUpdate(MonitorTaskInfo monitorTaskInfo) {
     LOG.info("Execute monitor update {}", monitorTaskInfo);
+    int jobRetentionDays = monitorTaskInfo.getDefaultRetentionDays();
     try {
+      // Mark expired task as TIMEOUT
```

Comment:
Add more info why we add this.
Also I remember we want to clean up the RUNNING tasks when worker starts. Is that fix already in?

Reply:
Added the logic in the next commit.